core/serialization/blocks.js

```diff
@@ -399,8 +399,8 @@ const loadPrivate = function(
  * @param {!State} state The state object to reference.
  */
 const loadCoords = function(block, state) {
-  const x = state['x'] === undefined ? 10 : state['x'];
-  const y = state['y'] === undefined ? 10 : state['y'];
+  const x = state['x'] === undefined ? 0 : state['x'];
+  const y = state['y'] === undefined ? 0 : state['y'];
```

Comment:
I think paste has a default of 10, just curious why this is different?

Reply:
I thought the same thing! But this messes with the alignment of the flyout :/

And thinking about it now, I think it makes more sense that if coordinates aren't provided the block should just go to `(0, 0)`. Less likely to cause bugs that way I think.